slack-base/src/main/java/com/hubspot/slack/client/models/events/channel/SlackChannelArchiveEventIF.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.slack.client.models.events.channel;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.events.SlackEvent;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+@JsonDeserialize(as = SlackChannelArchiveEvent.class)
+public interface SlackChannelArchiveEventIF extends SlackEvent {
+  @JsonProperty("channel")
+  String getChannelId();
+
+  @JsonProperty("user")
+  String getUserId();
+
+  default String getTs() {
```

Comment:
I think it'd be clearer here if we had the `@Override` annotation on these methods